libs/util-constants/src/lib/units.ts

```diff
@@ -0,0 +1 @@
+export const UNIT_TYPES = { COMPONENT: '0', AGENT: '1' } as const;
```

Comment:
we also have the `TOKENOMICS_UNIT_TYPES` in registry app, do you think we can have one source of truth? maybe we can keep rename this one to `TOKENOMICS_UNIT_TYPES`, remove old one from the app itself and update imports